modules/openapi-json-schema-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -455,7 +455,10 @@ public void processOpts() {
                 (openAPI.getComponents() != null ? openAPI.getComponents().getSecuritySchemes() : null) : null;
         List<CodegenSecurity> authMethods = fromSecurity(securitySchemeMap);
         if (ProcessUtils.hasHttpSignatureMethods(authMethods)) {
-            supportingFiles.add(new SupportingFile("signing." + templateExtension, packagePath(), "signing.py"));
+            supportingFiles.add(new SupportingFile("signing_http_signing." + templateExtension, packagePath() + File.separatorChar + "signing", "http_signing.py"));
+        }
+        if (ProcessUtils.hasAwsSignatureV4Methods(authMethods)) {
+            supportingFiles.add(new SupportingFile("signging_aws_sigv4." + templateExtension, packagePath() + File.separatorChar + "signing", "aws_sigv4.py"));
```

Comment:
How about also adding an init file to the signing folder?

Reply:
I had added the template, but forget to add it as a supporting file! Good catch!